src/lib/constants.js

```diff
@@ -1,5 +1,4 @@
 export const COLUMN_COUNT_QUERY_KEY = 'columns';
 export const WINDOW_STORAGE_NAMESPACE = 'music-markdown';
-export const REPOS_RESOURCE = 'repos';
-export const REPO_REGEX = `${REPOS_RESOURCE}/:repo([^/]+/[^/]+)`;
+export const REPO_REGEX = `repos/:repo([^/]+/[^/]+)`;
```

Comment:
When I'd suggested that we remove REPOS_RESOURCE, I had actually meant that we drop the word "repos" from everywhere in the code. The only reason it was there in the first place was because I was trying to match the github API. Now that we don't match the github API, the word itself in the URL, and by extension the constant, is noise. Sorry I wasn't clear. 

I'm okay with keeping it though, but if we do, we shouldn't have `repos/` as part of the REPO_REGEX because it makes the links and the routes asymmetric. For example:

Link: /viewer/repos/${repo}/${branch}/${item.path}
Route: /viewer/${REPO_REGEX}/:branch/:path*

A person unfamiliar with the code would wonder why 'repos' is part of the link but isn't part of the route.

Reply:
Agreed, that's what I was trying to avoid in the first place by having the constant.